test/e2e/tests/helper.py

```diff
@@ -55,4 +56,57 @@ def get_certificate(self, ca_arn: str, cert_arn: str):
             assert certificate is not None
             return certificate
         except self.acmpca_client.exceptions.ClientError:
+            pass
+
+    def create_root_ca(self):
```

Comment:
Can we create the root certificate using ACK?

Reply:
This method isn't being used, removing it